Sources_v3/StreamChatUI/Common Views/ChatSquareButton.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatSquareButton: Button {
+    open var defaultSize: CGFloat?
+    
+    public init(image: UIImage? = nil, tintColor: UIColor? = nil) {
+        super.init(frame: .zero)
+        setImage(image, for: .normal)
+        self.tintColor = tintColor
+    }
+    
+    override public func defaultAppearance() {
+        defaultSize = 20
+    }
+    
+    override open func setUpAppearance() {
+        imageView?.contentMode = .scaleAspectFit
+    }
```

Comment:
`setUpAppearance` should be empty https://www.notion.so/UIConfig-and-View-Structure-bdbd703e4ba44e1c8039b43ccf17f56f